classes/publication/maps/Schema.php

```diff
@@ -155,7 +155,7 @@ function ($citation) {
                     $output[$prop] = $retVal;
                     break;
                 case 'fullTitle':
-                    $output[$prop] = $publication->getFullTitles();
+                    $output[$prop] = $publication->getFullTitles('html');
```

Comment:
@NateWr, just in case you haven't flagged it, the changed interpretation of title fields from plaintext to HTML in API interactions should probably be included in the release notebook.

Reply:
Thanks! Flagged at https://github.com/pkp/pkp-docs/issues/1059

Reply:
Also, can we add a new property to the publication's return value here that's like `fullTitleText`? That way, someone who is using the API can get either version of the title.